fonts.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+  font-family: "Montserrat";
+  src: url("fonts/montserrat-regular-webfont.woff") format("woff");
+  src: url("fonts/montserrat-regular-webfont.woff2") format("woff2");
+  font-weight: normal;
+  font-style: normal; }
+
+@font-face {
+  font-family: "Montserrat";
+  src: url("fonts/montserrat-medium-webfont.woff") format("woff");
+  src: url("fonts/montserrat-medium-webfont.woff2") format("woff2");
+  font-weight: 500;
+  font-style: normal; }
+
+@font-face {
+  font-family: "Montserrat";
+  src: url("fonts/montserrat-semibold-webfont.woff") format("woff");
+  src: url("fonts/montserrat-semibold-webfont.woff2") format("woff2");
+  font-weight: 600;
+  font-style: normal; }
+
+@font-face {
+  font-family: "Montserrat";
+  src: url("fonts/montserrat-bold-webfont.woff") format("woff");
+  src: url("montserrat-bold-webfont.woff2") format("woff2");
+  font-weight: bold;
+  font-style: normal; }
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6002411)